internal/librarian/generate.go

```diff
@@ -192,9 +192,10 @@ func (r *generateRunner) runGenerateCommand(ctx context.Context, outputDir strin
 		if libraryID == "" {
 			return "", errors.New("bug in Librarian: Library not found during generation, despite being found in earlier steps")
 		}
+		generateRequest := filepath.Join(r.repo.Dir, config.LibrarianDir, config.GenerateRequest)
```

Comment:
I think this can be figured out in the docker package too. I think the value should be in the outputDir, not in the repo.

Reply:
I thought the `generate-request.json` should live in `.librarian` directory, which is at the root of the language repo.

Reply:
this is a just temporary file that is never persisted. It is only used to communicate to-from the container contract.